lib/src/container/encapsulate.rs

```diff
@@ -118,13 +118,13 @@ fn build_oci(
 
     labels.insert(OSTREE_COMMIT_LABEL.into(), commit.into());
 
-    for (k, v) in config.labels.iter().map(|k| k.iter()).flatten() {
+    for (k, v) in config.labels.iter().flat_map(|k| k.iter()) {
         labels.insert(k.into(), v.into());
     }
     // Lookup the cmd embedded in commit metadata
     let cmd = commit_meta.lookup::<Vec<String>>(ostree::COMMIT_META_CONTAINER_CMD)?;
     // But support it being overridden by CLI options
-    let cmd = config.cmd.as_ref().or_else(|| cmd.as_ref());
+    let cmd = config.cmd.as_ref().or(cmd.as_ref());
```

Comment:
Clippy is not liking this change.

Reply:
Ah awesome, old clippy doesn't like new code, new clippy doesn't like old code.  Well, we can deal with this one when we update the CI pinned clippy.

Reply:
Took this to https://github.com/rust-lang/rust-clippy/pull/7639#issuecomment-1050340564